monai/transforms/transforms.py

```diff
@@ -441,10 +441,45 @@ def __call__(self, img):
         return data
 
 
-# if __name__ == "__main__":
-#     img = np.array((1, 2, 3, 4)).reshape((1, 2, 2))
-#     rotator = RandRotate90(prob=0.0, max_k=3, axes=(1, 2))
-#     # rotator.set_random_state(1234)
-#     img_result = rotator(img)
-#     print(type(img))
-#     print(img_result)
+@export
+class RandomRotate(Randomizable):
+    """Randomly rotates the input arrays.
+
+    Args:
+        prob (float): Probability of rotation.
+        angle (float): Rotation angle in degrees.
```

Comment:
I checked torchvision's "RandomRotation":
https://pytorch.org/docs/master/torchvision/transforms.html#torchvision.transforms.RandomRotation
I think it's different from **RandomFlip** that we should **randomly pick 1 angle** value to rotate instead of a fixed angle.
Should be same logic as your "RandomZoom" transform.
What do you think?
Thanks.

Reply:
Makes sense. Changed to use range just like albumentations and torchvision.